docs/SceneWalkthrough/Auth&FriendsWalkthrough.md

```diff
@@ -0,0 +1,7 @@
+## **Auth & Friends Demo**
+This demo showcases an implementation of the friends list into a game UI. The panel on the right shows the user's friend list and status of each friend.
+- The ``Refresh`` button refreshes the list, repolling for information.
+- The ``Friends Overlay`` button opens the EOS overlay.
+
```

Comment:
a note that you can open and close the friends tab would be nice and applicable to other scenes

Reply:
i think this needs like a 
- The ``Friends`` tab, on the top left of the friends list, opens and closes the friends list. **In other samples it may start closed**